src/components/Calculator.js

```diff
@@ -1,41 +1,139 @@
 import React, { Component } from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Issue raised by this disabled rule can be solved and it seems you have already solved it. The solution is adding a `constructor` to the Calculator class. Please re-enable this rule and see if the warning persists. I believe it shouldn't.